services/data/methods.go

```diff
@@ -1053,22 +1051,19 @@ func (d *dataClient) WriteTickets(tickets []*types.Ticket) error {
 		if err != nil {
 			return err
 		}
-	}
-	if len(tickets) > 0 {
-		datadog.Info("Wrote tickets %v", tickets)
+		datadog.Info("Wrote ticket (ID, Summary, AssigneeName) %v", ticket.ID, ticket.Summary, ticket.AssigneeName)
 	}
 	return nil
 }
 
 func (d *dataClient) UpdateTickets(tickets []*types.Ticket) error {
 	for i := range tickets {
-		_, err := d.Client.Update(tickets[i])
+		ticket := tickets[i]
+		_, err := d.Client.Update(ticket)
 		if err != nil {
 			return err
 		}
-	}
-	if len(tickets) > 0 {
```

Comment:
You're doing this outside of the loop, so it'll only send an event for the last ticket, even though this updates N tickets.
Maybe we don't want to send N events, but this is just sending partial data... How about we concatenate all the ticket update logs into one string and send that as one event?